src/explore.c

```diff
@@ -34,18 +38,56 @@
 #define PRETTY_GREY   "\e[0;37m"
 #define PRETTY_SELECT "\e[46m"
 
+#define PRETTY_PRUNE  "\e[45m"
+
 #define PRETTY_CYAN "\e[0;36m"
 #define PRETTY_NORM "\e[0m"
 
 #define CURSOR_DISABLE "\e[?25l"
 #define CURSOR_ENABLE  "\e[?25h"
 
+#define MARK_PRUNE  "insert into cmdan(hash, type, desc) "       \
+                    "values (\"%s\", %d, \"%s\") "               \
+                    "on conflict(hash) do update "               \
+                    "set type=excluded.type, desc=excluded.desc" \
+                    ";"
```

Comment:
This command adds the idea of an annotation table - right now the only annotation supported is "prune" (a value of `1`). Annotation descriptions are unused so far